src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        String colorOfFigure = colorSupplier.getRandomColor();
+        int numberOfFigure = random.nextInt(6);
```

Comment:
first of all - `6` is a magic number, you should define a constant instead

apart from that, you actually have `5` figures :)